docs/content/docs/build/container-states-events.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Container states and events
+menuPosition: 2
+author: rick-kirkham
+---
+
+This article provides a detailed description of the lifecycle states of the containers and container events. It assumes that you are familiar with [Containers](./containers.md).
+
+{{< callout note >}}
+
+In this article the term "creating client" refers to the client on which a container is created. 
+When it is important to emphasize that a client is *not* the creating client, it is called a "subsequent client". 
+It is helpful to remember that "client" does not refer to a device or anything that persists between sessions with your application. 
+When a user closes your application, the client no longer exists: a new session is a new client. 
+So, when the creating client is closed, there is no longer any creating client. 
+The device is a subsequent client in all future sessions.
+
+{{< /callout >}}
+
+There are four types of states that a container can be in. Every container is in exactly one state of each of these types. Details and diagrams are below.
+
+- **Publication status**: 
+
+    - unpublished
+    - publishing (to the Fluid service)
+    - published
+    - deleted (from the service)
+
+- **Synchronization status**: 
+
+    - dirty (on a given client)
+    - saved
+
+- **Connection status**: 
+
+    - disconnected (from the Fluid service)
+    - establishing connection (to the service)
+    - catching up (with data from other clients)
+    - connected
+
+- **Local readiness status**: 
+
+    - ready
+    - disposed (on the client)
+
+### Publication status states
+
+Publication status refers to whether or not the container has been *initially* saved to the Fluid service, and whether it is still persisted there. 
+Think of it as a mainly *service-relative* state because it is primarily about the container's state in the Fluid service and secondarily about its state on the creating client. The following diagram shows the possible publication states and the events that cause a state transition. Details are below the diagram.
+
+<!-- TO MODIFY THIS DIAGRAM, SEE INSTRUCTIONS AT THE BOTTOM OF THIS FILE. -->
+![A state diagram of the four possible publication states](./images/PublicationStates.svg)
```

Comment:
To fix the image links, do the following:

1. Create a new folder called container-states-events under the build folder.
2. Move the container-states-events.md file into that folder and rename it to index.md
3. Move the images folder under the container-states-events folder.

The links should start working after that.